techniques/general/G171.html

```diff
@@ -51,92 +49,88 @@
-   </section><section id="examples"><h2>Examples</h2>
-      <section class="example" id="example-1"><h3>Example 1</h3>
+<section id="description"><h2>解説</h2>
+      <p>このテクニックの目的は、ウェブコンテンツにおける音声の使用を利用者が制御できるようにすることである。スクリーンリーダーを使う利用者にとっては、ウェブコンテンツから音声が再生されていると、スクリーンリーダーの音声を聞き取る上で非常に気が散ったり聞き取るのが困難になるかもしれない。利用者の要求によってのみ音声を再生する方法を提供することで、利用者はスクリーンリーダーからの出力と干渉せずに音声を聴くために必要な制御を行えるようになる。</p>
+   </section><section id="examples"><h2>事例</h2>
+      <section class="example" id="example-1"><h3>事例 1</h3>
          
-            <p>A web page from a grey whale conservation society has a looping background sound of grey whales singing. There are also sounds of water splashing. The sounds do not start automatically. Instead, the Web content provides a link at the top of the page to allow the user to start the sounds manually. The button says "Turn sounds on." After pressing the "turn sounds on" button, the sounds are heard. The user is then presented with an option to "turn sounds off."</p>
+            <p>あるコククジラ保護団体のウェブページでは、コククジラの鳴き声と水しぶきの音が背景として繰り返し流れるようになっている。その音声は自動的には開始されない。その代わり、ウェブコンテンツには先頭付近に、利用者が手動で再生を開始できるボタンが置かれている。ボタンには「音声を再生」と書かれており、このボタンを押すことで音声が聞こえてくる。再生が始まると利用者には「音声を停止する」という選択肢が提供される。</p>
          
       </section>
-      <section class="example" id="example-2"><h3>Example 2</h3>
+      <section class="example" id="example-2"><h3>事例 2</h3>
          
-            <p>A link is provided to a sound file that includes the sounds of the grey whales. The link text says, "Hear the song of the grey whale (mp3)."</p>
+            <p>コククジラの音が含まれる音声ファイルへのリンクが提供されている。リンクテキストには「コククジラの鳴き声を聴く (MP3)」と書かれている。</p>
          
       </section>
    </section>
-<section id="related"><h2>Related Techniques</h2><ul>
+<section id="related"><h2>関連テクニック</h2><ul>
       <li><a href="../general/G60">G60: Playing a sound that turns off automatically within three seconds</a></li>
       <li><a href="../general/G170">G170: Providing a control near the beginning of the web page that turns off sounds that play automatically</a></li>
    </ul></section>
-<section id="tests"><h2>Tests</h2>
-      <section class="procedure" id="procedure"><h3>Procedure</h3>
+<section id="tests"><h2>検証</h2>
+      <section class="procedure" id="procedure"><h3>手順</h3>
          <ol>
-            <li>Load a web page that is known to contain sounds that play for 3 seconds or longer.</li>
-            <li>Check that no sounds play automatically.</li>
-            <li>Check that there is a way for a user to start sounds manually.</li>
+            <li>3 秒以上の音声を再生することが既知のウェブページを読み込む。</li>
+            <li>音声が自動的に再生されないことを確認する。</li>
```

Comment:
check that ... の訳を他のテクニックと合わせました。
```suggestion
            <li>音声が自動的に再生されないことをチェックする。</li>
```